### PR TITLE
TINKERPOP-2905 Always pass `logHandler` to transporter

### PR DESCRIPTION
**Pull request by disq**

### CODE REVIEW

File: gremlin-go/driver/transporterFactory.go [hunk 1]
```diff
@@ -31,11 +31,17 @@ const (
...
 
-func getTransportLayer(transporterType TransporterType, url string, connSettings *connectionSettings) (transporter, error) {
+func getTransportLayer(transporterType TransporterType, url string, connSettings *connectionSettings, logHandler *logHandler) (transporter, error) {
 	var transporter transporter
 	switch transporterType {
 	case Gorilla:
-		transporter = &gorillaTransporter{url: url, connSettings: connSettings, writeChannel: make(chan []byte, writeChannelSizeDefault), wg: &sync.WaitGroup{}}
+		transporter = &gorillaTransporter{
+			url:          url,
+			logHandler:   logHandler,
```
Comment (disq) [inline gremlin-go/driver/transporterFactory.go:40]: The whole block was reformatted and `logHandler` was added.

**Comment by vkagamlyk** [comment]
@disq thank you for fixing this!

VOTE +1

**Comment by Cole-Greer** [comment]
Thanks @disq, this looks good to me. Just so you know what happens next, our review process requires 3 committers to give the PR a VOTE +1 in the comments here (Or single VOTE +1 with a 7 day waiting period). Non-committers can give non-binding votes for changes they support. Once the change is approved one of the committers will merge the PR into the appropriate branches. The specifics are all defined [here](https://tinkerpop.apache.org/docs/current/dev/developer/#rtc).

VOTE +1 (non-binding)

**Comment by kenhuuu** [comment]
VOTE +1

**Pull request merged**